patches/LICENSE

```diff
@@ -0,0 +1,340 @@
+		    GNU GENERAL PUBLIC LICENSE
```

Comment:
Why are the patches using the GPL?  They should be Apache licensed, like the rest of the repos.

Reply:
The license from the patches directory should match the LICENSE of the project we're porting. We do this because the patches are diffs and can contain code from the original tool.
We have some documentation on it here: https://github.com/ZOSOpenTools/meta/blob/main/CONTRIBUTING.md#coding-guidelines .